apps/aefate/Makefile

```diff
@@ -0,0 +1,7 @@
+.PHONY: all
+all: fate_eval
+
+.PHONY: fate_eval
+fate_eval:
```

Comment:
Any reasons why PHONY and not the actual file, with prerequisites?

Reply:
The prerequisites would be source files in an outside dependency: `aebytecode`. Thus, I made it  `PHONY`.